### PR TITLE
updated connecting to SC section

### PR DESCRIPTION
**Pull request by SeanFeldman**
For a long time the examples and guidance was MSMQ specific.
Changed it to be MSMQ agnostic and in examples state that it's for MSMQ.

@Particular/documentation please review

### CODE REVIEW

**Comment by SimonCropp** [comment]
@gbiellem or @mikeminutillo can u review and merge

**Comment by gbiellem** [comment]
I think we need to clarify that the ServiceControl queue name is based on the service name as well.  
That not obvious in this doco but that's unrelated to what @SeanFeldman did.  I'll add it to this PR then i think we can merge

File: servicecontrol/plugins/index.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ NOTE: For NServiceBus version-dependent requirements for each plugin, review the
 Once deployed on an active endpoint, the endpoint sends plugin-specific information to ServiceControl. Plugins send messages using the defined endpoint transport to the ServiceControl queue. Location of ServiceControl queue is determined by the following:
 
 1. **Endpoint`s configuration file** 
-Check for an `appSetting` named `ServiceControl/Queue` e.g. `<add key="ServiceControl/Queue" value="particular.servicecontrol@machine"/>`.
+Check for an `appSetting` named `ServiceControl/Queue` e.g. `<add key="ServiceControl/Queue" value="particular.servicecontrol"/>`.
 1. **Convention based on the configured Error Queue machine** 
-If an error queue is configured, for example `error@MachineName`, then the queue `Particular.ServiceControl@MachineName` will be used.
+If an error queue is configured, for example `error@MachineName` with MSMQ transport, then the queue `Particular.ServiceControl@MachineName` will be used.
```
Comment (mikeminutillo) [inline servicecontrol/plugins/index.md]: This sentence is confusing. Does this mean that if we have a configured error queue that includes a machine name then we will assume the Service Control queue is on the same machine? What if the error queue is just a queue name (i.e. current machine?). What if we are using a transport that does not include machine name (i.e. RabbitMQ)?

File: servicecontrol/plugins/index.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ NOTE: For NServiceBus version-dependent requirements for each plugin, review the
...
 1. **Convention based on the configured Audit Queue machine** 
-If an audit queue is configured, for example `audit@MachineName`, then the queue `Particular.ServiceControl@MachineName` will be used.
+If an audit queue is configured, for example `audit@MachineName` with MSMQ transport, then the queue `Particular.ServiceControl@MachineName` will be used.
```
Comment (mikeminutillo) [inline servicecontrol/plugins/index.md]: Are these conventions applied in this order? If I have audit and error going to different machines, where do the ServiceControl messages go?

Comment (gbiellem) [inline servicecontrol/plugins/index.md]: Yes they are applied in that order -  https://github.com/Particular/ServiceControl.Plugin.Nsb4.Heartbeat/blob/master/src/ServiceControl.Plugin.Nsb4.Heartbeat/ServiceControlBackend.cs#L86

File: servicecontrol/plugins/index.md (hunk 1 above)
Comment (gbiellem) [inline servicecontrol/plugins/index.md]: @mikeminutillo I've seen this in other docs too.
@SimonCropp What's the convention for queue names generally within docs? Is this right or is there something more generic?

Comment (SimonCropp) [inline servicecontrol/plugins/index.md]: we r not consistent with that yet. so i would not block this PR for that

**Pull request merged**